pkgs/development/python-modules/aws-google-auth/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, buildPythonPackage, fetchPypi, beautifulsoup4, boto3, configparser, keyring, keyrings-alt, lxml, pillow, requests, six, tabulate, tzlocal, nose, mock }:
+
+buildPythonPackage rec {
+  version = "0.0.29";
+  pname = "aws-google-auth";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1q587v2vnnzcybwryrlnyji0rv5wvkkd0dccna30jkc13gkic31f";
+  };
+
+  propagatedBuildInputs = [ beautifulsoup4 boto3 configparser keyring keyrings-alt lxml pillow requests six tabulate tzlocal ];
+  checkInputs = [ mock ];
+
+  preCheck = ''
+    export HOME=$TMPDIR
+  '';
+
+  # PyPI tarball is missing some test files
+  doCheck = false;
```

Comment:
They you should probably use `fetchFromGitHub` to get the source from the github repo which has all of the tests.